scripts/codegen.ts

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env zx
+import { convertCase } from '@axelarjs/utils/case-conversion';
+import { capitalize } from '@axelarjs/utils/string';
+
+import fs from 'fs/promises';
```

Comment:
Can this file be imported from somewhere? We'd wanna avoid duplication across repos. Maybe we check this into `gmp-sdk` repo?

Reply:
The whole codegen utility?

Reply:
If so, the most natural place would be in our evm package in the axelarjs monorepo since other repos aren't ts-native. 

I'm also fine just leaving this here for now as it's an implementation detail in a build script that can be changed later, and won't affect other repos.